app/views/layouts/_user_menu.html.haml

```diff
@@ -16,6 +16,11 @@
     = link_to(conference_program_tracks_path(@conference.short_title)) do
       %span.fa.fa-road
       My Tracks
+-if @conference && current_user.booths.where(conference_id: @conference.id).count > 0
```

Comment:
For proposals, we show the link even if the user doesn't have any submissions, so that the user can go to the index page and add a new proposal from there. Did you check this (how it is done in other places of the app)?
Which way do you think we should do it?